en/dev_log/ulog_file_format.md

```diff
@@ -73,8 +77,8 @@ struct message_header_s {
 };
 ```
 
-* `msg_size` is the size of the message in bytes without the header
-* `msg_type` defines the content, and is a single alphabet character.
+- `msg_size` is the size of the message in bytes without the header.
+- `msg_type` defines the content, and is a single alphabet character.
 
 :::note
 Message types described below are noted with the single alphabet `msg_type` characteristic in the beginning.
```

Comment:
Is this clear enough for the readers to recognize that the: `"'B': Flag Bits Message"` indicates that the `msg_type` of that message corresponds to the character `'B'`?

Because this is one of the first things I was so confused about the document, as it wasn't so clear what all those 'B', 'f', 'I', 'M', etc. meant :thinking:

Reply:
No. We need to make that clear right up the top of the doc somewhere. I had same question and forgot to raise it.

Reply:
I added a [comment](https://github.com/PX4/PX4-user_guide/pull/1887#discussion_r876820169) about this, where the proposed change will expose the `msg_type` character clearly in the struct definition!